bindings/java/iota-wallet-java/lib/build.gradle

```diff
@@ -58,11 +72,15 @@ jar {
         include '*.so', '*.dylib', '*.dll'
     }
 
+    from('native/target/' + findProperty('buildTarget') + '/release') {
```

Comment:
Is this needed together with the one above?

Reply:
Yes this is needed for the aarch64-apple-darwin target release